Makefile

```diff
@@ -35,8 +35,8 @@ deploy-helm: image ## Deploys image with helm
 	--set image="$(IMAGE):$(TAG)",imagePullPolicy="$(PULL)"
 
 deploy-openshift: image ## Deploys image to openshift
-	oc get project bdc || oc new-project bdc
-	oc process -f openshift/dataverse-broker.yaml -p IMAGE=$(IMAGE):$(TAG) | oc apply -f -
+	oc project bdc || oc new-project bdc
```

Comment:
Here, the Makefile will switch you to the correct namespace on successful "oc project bdc"